data-raw/make_cluster_vcov.R

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
I also think this would be cool to add to a vignette, maybe the main one, maybe a secondary one.